src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -892,6 +897,24 @@ public GcpBigQueryDatasetResource createControlledBigQueryDataset(
         "Error creating controlled BigQuery dataset in the workspace.");
   }
 
+  /** Used by tests only. */
+  public TerraWorkspaceResource createReferencedTerraWorkspace(
```

Comment:
I'm still uncomfortable about having an asymetrical commands for datasource. this means before the UI have datasource, this command is useless in the CLI. i'm fine with this going in but i'd like Jenn's opinion on the CLI interface.